src/check_cve/check.py

```diff
@@ -65,23 +70,51 @@ def is_vulnerable(host):
 def main():
     """Parse arguments, handle logging, and call vulnerability check."""
     args = docopt.docopt(__doc__, version=__version__)
-    # Set up logging
-    log_level = args["--log-level"]
+
+    # Validate and convert arguments as needed
+    schema = Schema(
+        {
+            "--log-level": And(
+                str,
+                Use(str.lower),
+                lambda n: n in ("debug", "info", "warning", "error", "critical"),
+                error="Possible values for --log-level are "
+                + "debug, info, warning, error, and critical.",
+            ),
+            "--retries": And(
+                Use(int),
+                lambda n: 0 <= n,
+                error="--retries must be an integer greater than 0.",
```

Comment:
Your error string says "greater than 0" but your lambda checks if it is greater than _or equal to_ zero.